resources/views/device-group/index.blade.php

```diff
@@ -57,6 +57,39 @@ class="fa fa-trash" aria-hidden="true"></i></button>
             </div>
         </div>
     </div>
+    <div class="container-fluid">
```

Comment:
Don't start a new container, just add it to the existing one.

Reply:
changed